test/model.test.js

```diff
@@ -1028,7 +1062,6 @@ describe.onServer('Remote Methods', function() {
           return User.login(CREDENTIALS);
         }).then(function(token) {
           accessToken = token;
-          userId = token.userId;
```

Comment:
This change seems rather unrelated, could you please remove it (revert this line)?